### DISCUSSION TITLE
Difference in reported file size

### DISCUSSION THREAD

**Discussion by rmeske** [iterate-ch/cyberduck#15903] [Q&A] [answered]
I am using Cyberduck to access AWS S3 and have noticed a difference in reported file size between the two.  At first I ignored it attributing it to how S3 stores the files in an encrypted bucket.  However, as the files get into the gigabyte size the difference becomes much larger.  For example, I have a file that shows 12.5gb in the S3 console and 13.4gb in Cyberduck.

The bucket is encrypted and is not versioned.

Can anyone explain why the difference in file size reporting?

**Comment by tatze96** [comment, marked as answer]
The unequal file sizes are likely caused by the use of differing data units.

In this case, Cyberduck uses Gigabyte as its data unit, while the S3 console likely uses Gibibytes. 13,4 Gigabyte equals 12,47 Gibibytes.

**Reply by rmeske** [reply]
Thank you.  That makes sense.  It would be nice to be able to toggle the unit that Cyberduck uses.
